src/component/utils/utils.js

```diff
@@ -0,0 +1,28 @@
+import { patchArticleDownvote, patchArticleUpvote } from "./api";
+
+
```

Comment:
Similar to above, to avoid two functions that are identical apart from what the `setUserUpvote` state is set to, you could have one function that takes in a value as an argument which determines what the state is set to.

Reply:
Done white_check_mark